app/assets/config/manifest.js

```diff
@@ -2,3 +2,5 @@
 //= link_directory ../stylesheets .css
 //= link_tree ../../javascript .js
 //= link_tree ../../../vendor/javascript .js
+//= require moment
+//= require bootstrap-datetimepicker
```

Comment:
Check this once, the app is only running after removing moment & bootstrap-datetimepicker